lib/tests/streamlit/caching/common_cache_test.py

```diff
@@ -22,19 +22,19 @@
 
 import streamlit as st
 from streamlit.caching.cache_errors import CacheReplayClosureError
-from streamlit.scriptrunner import script_run_context
+from streamlit.runtime.scriptrunner import script_run_context
```

Comment:
nit: can be merged into the `from streamlit.runtime.scriptrunner import...` below